### PR TITLE
chore: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by palpatim**
Add @aws-amplify/developer-advocates as CODEOWNERS for all paths

_Issue #, if available:_

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by timngyn — APPROVED** [review]
Looks good to me!!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+/src/pages/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/** @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/android @aws-amplify/amplify-android @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/flutter @aws-amplify/amplify-flutter @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/ios @aws-amplify/amplify-ios @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/**/js  @aws-amplify/amplify-js @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/cli @aws-amplify/amplify-cli @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/fragments/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
```
Comment (timngyn) [inline .github/CODEOWNERS:13]: Just noticed that, the `guides` routes already has `aws-amplify/developer-advocates` :
- `/src/pages/guides`
- `/src/fragments/guides`

**Review by katiegoines — CHANGES_REQUESTED** [review]
@palpatim - Lines 5 and 13 list `@aws-amplify/developer-advocates` twice. May I remove the duplicate? Thanks!

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-/src/fragments/**/ios @aws-amplify/amplify-ios @aws-amplify/amplify-support-eng
-/src/fragments/**/js  @aws-amplify/amplify-js @aws-amplify/amplify-support-eng
-/src/fragments/cli @aws-amplify/amplify-cli @aws-amplify/amplify-support-eng
-/src/fragments/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng
-/src/fragments/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng
+* @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/** @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/cli @aws-amplify/amplify-cli @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/console @aws-amplify/amplify-hosting @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
+/src/pages/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng @aws-amplify/developer-advocates
```
Comment (katiegoines) [inline .github/CODEOWNERS:5]:
```suggestion
/src/pages/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (katiegoines) [inline .github/CODEOWNERS:13]:
```suggestion
/src/fragments/guides @aws-amplify/developer-advocates @aws-amplify/amplify-support-eng
```

**Review by katiegoines — APPROVED** [review]

**Pull request closed without merging**